### ISSUE TITLE
re-implementation with tensorflow cannot achieve the same accuracy on mxnet.

### ISSUE THREAD

**Issue by Jar7** [luckycallor/InsightFace-tensorflow#27]
Hi, I apply insightface for face recognition with my own dataset by simply add 2 fc layers and output the index of the class.
Firstly I trained it on mxnet, fixed the feature network(author provided pretrained model: r100), only finetune that 2 additional fc layers. Then I re-implemented it with tensorflow, the parameters and optimizer and initialization is the same. However, I cannot reproduce the same result as on mxnet.
Any suggestion, please? Is there any detail could I ignored?
Thank you.

**Comment by clhne** [comment]
If the accuracy on  mxnet is normal, so  why TF？
